soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -292,6 +292,19 @@ static bool ItemHandler(std::shared_ptr<Ship::Console> Console, const std::vecto
     return CMD_SUCCESS;
 }
 
+static bool GiveItemHandler(std::shared_ptr<Ship::Console> Console, const std::vector<std::string>& args) {
+    if (args.size() != 2) {
+        SohImGui::console->SendErrorMessage("[SOH] Unexpected arguments passed");
+        return CMD_FAILED;
+    }
+
+    Player* player = GET_PLAYER(gGlobalCtx);
+    player->getItemId = std::stoi(args[1]);
+    func_8002F434(&player->actor, gGlobalCtx, std::stoi(args[1]), 30.0f, 40.0f);
```

Comment:
is there a specific reason to use this over `GiveItemWithoutActor`?
https://github.com/HarbourMasters/Shipwright/blob/86faa42acf34d988098809a1d0e01b9fe1d051c9/soh/src/code/z_actor.c#L1958